e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -214,19 +213,14 @@ $intermediateCert1CertPath = "$PSScriptRoot/intermediateCert1.cer";
 $intermediateCert2CertPath = "$PSScriptRoot/intermediateCert2.cer";
 $verificationCertPath = "$PSScriptRoot/verification.cer";
 
-$groupCertCommonName = "xdevice1";
-$groupPfxPath = "$PSScriptRoot/Group.pfx";
-$groupCertChainPath = "$PSScriptRoot/GroupCertChain.p7b";
+$iotHubX509DeviceCertCommonName = "iothubx509device1";
+$iotHubX509DevicePfxPath = "$PSScriptRoot/IotHubX509Device.pfx";
+$iotHubX509CertChainDeviceCommonName = "iothubx509chaindevice1";
+$iotHubX509ChainDevicPfxPath = "$PSScriptRoot/IotHubX509ChainDevice.pfx";
 
-$iotHubCertCommonName = "iothubx509device1";
-$iotHubPfxPath = "$PSScriptRoot/IotHub.pfx";
-$iotHubCertChainDeviceCommonName = "iothubx509chaindevice1";
-$iotHubChainDevicPfxPath = "$PSScriptRoot/IotHubChainDevice.pfx";
-
-# Extra/removed/deleted
-$deviceCertCommonName = "iothubx509device1";
-$individualDeviceCertPath = "$PSScriptRoot/Device.cer";
-$individualDevicePfxPath = "$PSScriptRoot/Device.pfx";
```

Comment:
X509 self-signed certificates are being generated on-demand, so this can be removed